.github/workflows/source/travis_matrix.py

```diff
@@ -23,4 +23,4 @@
 print("Tests in initial list but not in the matrix:")
 print(list(set(all_tests) - set(matrix_elements)))
 
-assert( matrix_elements == all_tests )
```

Comment:
I guess we test some things twice now, because this test failed while the prints above listed `[]` and `[]`.